nautobot/docs/release-notes/version-2.2.md

```diff
@@ -44,3 +52,117 @@ The CSV import functionality for all models has been changed from a synchronous
 
 !!! tip
     Users now must have the `run` action permission for `extras > job` (specifically the `nautobot.core.jobs.ImportObjects` Job) in order to import objects, in addition to the normal `add` permissions for the object type being imported.
+
+#### Plugin to App Renames([#5341](https://github.com/nautobot/nautobot/issues/5341))
+
+`Installed Plugins` view has been renamed to `Installed Apps`. `Plugin` terminologies in `Installed Plugins` (now `Installed Apps`) view and dependent views have been changed to `App` throughout. `Plugin` references in documentation (excluding old release-notes) have been replaced by `App`. `Plugins` navigation menu has been renamed to `Apps`. See more details about this change in [#5353](https://github.com/nautobot/nautobot/pull/5353).
```

Comment:
I don't think the direct PR link is necessary - readers can follow the link to #5341 above and from there to the PR.

```suggestion
`Installed Plugins` view has been renamed to `Installed Apps`. `Plugin` terminologies in `Installed Plugins` (now `Installed Apps`) view and dependent views have been changed to `App` throughout. `Plugin` references in documentation (excluding old release-notes) have been replaced by `App`. `Plugins` navigation menu has been renamed to `Apps`.
```